geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ClientCommands.java

```diff
@@ -109,12 +107,10 @@ public Result listClient() {
       }
 
       String memberSeparator = ";  ";
-      Iterator<Entry<String, List<String>>> it = clientServerMap.entrySet().iterator();
 
-      while (it.hasNext()) {
-        Map.Entry<String, List<String>> pairs = (Map.Entry<String, List<String>>) it.next();
-        String client = (String) pairs.getKey();
-        List<String> servers = (List<String>) pairs.getValue();
+      for (Entry<String, List<String>> pairs : clientServerMap.entrySet()) {
+        String client = pairs.getKey();
+        List<String> servers = pairs.getValue();
         StringBuilder serverListForClient = new StringBuilder();
```

Comment:
Since we're already cleaning up this section of the code, it might be worth considering replacing 114-123 with a one liner: 

```
String serverListForClient = servers.stream().collect(Collectors.joining(memberSeparator));
```